analyze/plugins/tls-clienthello/ToDo.txt

```diff
@@ -0,0 +1,14 @@
+The following extensions are not checked:
```

Comment:
Does this mean the content is not checked? Or that the presence/order is not checked?

Reply:
Shoot... I have no idea....

Reply:
Removed.